python3-get-framework-k8s-api-version.spec

```diff
@@ -17,16 +17,24 @@
 
 
 %define upstream_name get-framework-k8s-api-version
+%define is_azure "%(sudo dmidecode -s bios-vendor | grep Microsoft)"
+%define is_google "%(sudo dmidecode -s bios-vendor | grep Google)"
```

Comment:
See comment in #2 we need a comment to clarify the limited applicability of these definitions, i.e. only works if installed in a running instance.